portfolio/src/test/java/com/google/sps/FindScheduleTest.java

```diff
@@ -0,0 +1,130 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class FindScheduleTest {
+  private static Instant BEGINNING_OF_DAY = Instant.parse("2020-06-25T00:00:00Z");
+  private static Instant END_OF_DAY = Instant.parse("2020-06-25T23:59:59Z");
+
+  private static long TIME_15_MINUTES = 900;
+  private static long TIME_30_MINUTES = 1800;
+  private static long TIME_45_MINUTES = 2700;
+  private static long TIME_60_MINUTES = 3600;
+
+  private static Instant TIME_0900 = BEGINNING_OF_DAY.plusSeconds(9 * TIME_60_MINUTES);
+  private static Instant TIME_1000 = BEGINNING_OF_DAY.plusSeconds(10 * TIME_60_MINUTES);
+  private static Instant TIME_1100 = BEGINNING_OF_DAY.plusSeconds(11 * TIME_60_MINUTES);
+  private static Instant TIME_1200 = BEGINNING_OF_DAY.plusSeconds(12 * TIME_60_MINUTES);
+  private static Instant TIME_1300 = BEGINNING_OF_DAY.plusSeconds(13 * TIME_60_MINUTES);
+  private static Instant TIME_1400 = BEGINNING_OF_DAY.plusSeconds(14* TIME_60_MINUTES);
+  private static Instant TIME_1500 = BEGINNING_OF_DAY.plusSeconds(15 * TIME_60_MINUTES);
+  private static Instant TIME_1600 = BEGINNING_OF_DAY.plusSeconds(16* TIME_60_MINUTES);
+  private static Instant TIME_1700 = BEGINNING_OF_DAY.plusSeconds(17 * TIME_60_MINUTES);
+  private static Instant TIME_1800 = BEGINNING_OF_DAY.plusSeconds(18 * TIME_60_MINUTES);
+
+
+  private final TaskPriority LOW_PRIORITY = new TaskPriority(1);
+
+  private FindSchedule schedule;
+
+  @Test
+  public void noTasksGreedy() {
+    Collection<CalendarEvent> events =
+        Arrays.asList(new CalendarEvent("Event 1", "2020-06-25T00:00:00Z", "2020-06-26T00:00:00Z"));
+    Collection<Task> tasks = Arrays.asList();
+
+    Collection<ScheduledTask> actual = FindSchedule.shortestTaskFirst(events, tasks, TIME_0900, TIME_1700);
+    Collection<ScheduledTask> expected = Arrays.asList();
+
+    Assert.assertEquals(expected, actual);
+  }
+
+  // TODO(tomasalvarez): rewrite all these tests to reflect changes made 
```

Comment:
Do you plan to fix these tests in this PR?

Reply:
I rewrote the tests after I refactored a lot of the code. I will remove the file from this PR and on the next PR the updated files will include all of the changes.